py/selenium/webdriver/chromium/webdriver.py

```diff
@@ -75,10 +75,9 @@ def launch_app(self, id):
     def get_network_conditions(self):
         """Gets Chromium network emulation settings.
 
-        :Returns:
-            A dict. For example:
-            {'latency': 4, 'download_throughput': 2, 'upload_throughput': 2,
-            'offline': False}
+        :Returns:     A dict. For example:     {'latency': 4,
```

Comment:
Personally prefer the previous layout... was easier to read.

Reply:
Tried to revert it but then the linter started to complain. I will leave it as it is.